GNUmakefile

```diff
@@ -77,10 +82,29 @@ teamcity-servicemap-generate:
 	cd tools/teamcity-generator;\
 		go run . --output $(OUTPUT_PATH) --version $(VERSION)
 
+
+clean-provider:
+	cd $(OUTPUT_PATH);\
+		go mod download;\
+		find . -type f -not -wholename "./.git*" -not -wholename "./.changelog*" -not -name ".travis.yml" -not -name ".golangci.yml" -not -name "CHANGELOG.md" -not -name "CHANGELOG_v*.md" -not -name "GNUmakefile" -not -name "docscheck.sh" -not -name "LICENSE" -not -name "README.md" -not -wholename "./examples*" -not -name ".go-version" -not -name ".hashibot.hcl" -print0 | xargs -0 git rm > /dev/null
+
+clean-tgc:
+	cd $(OUTPUT_PATH);\
+		rm -rf ./tfplan2cai/testdata/templates/;\
+		rm -rf ./tfplan2cai/testdata/generatedconvert/;\
+		rm -rf ./tfplan2cai/converters/google/provider;\
+		rm -rf ./tfplan2cai/converters/google/resources;\
+		find ./tfplan2cai/test/** -type f -exec git rm {} \; > /dev/null;\
```

Comment:
This needs the cai2hcl cleaning logic as well

Reply:
added